### ISSUE TITLE
Select element with long option text is forced outside the container when placed in a fieldset

### ISSUE THREAD

**Issue by kr8n3r** [alphagov/govuk-frontend#1264]
See here https://picturesque-chair.glitch.me/ on mobile (screeshot below)

I _believe_ this is because browsers apply `min-inline-size: min-content;` to the `fieldset`, which [MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/min-inline-size) says:
> ...defines the horizontal or vertical minimal size of an element's block, depending on its writing mode. It corresponds to either the min-width or the min-height property, depending on the value of writing-mode.

Applying `min-width: 0;` to `.govuk-fieldset` fixes it, but I haven't tested cross-browsers extensively (see screnshots at the end)

![picturesque-chair glitch me_(iPhone 5_SE) (1)](https://user-images.githubusercontent.com/3758555/55258087-c314d480-5259-11e9-9f06-c7922434134e.png)

Some tests:

iphone 6s
<img width="457" alt="Screen Shot 2019-03-29 at 19 42 41" src="https://user-images.githubusercontent.com/3758555/55258515-ef7d2080-525a-11e9-9b93-1b42076145cb.png">

iphone X
<img width="501" alt="Screen Shot 2019-03-29 at 19 44 16" src="https://user-images.githubusercontent.com/3758555/55258564-1cc9ce80-525b-11e9-9f26-35ebd5d25ced.png">

galaxy note 9
<img width="457" alt="Screen Shot 2019-03-29 at 19 45 05" src="https://user-images.githubusercontent.com/3758555/55258598-3703ac80-525b-11e9-9925-c99f397f32bb.png">

**Comment by colinrotherham** [comment]
@kr8n3r Thanks for the online demo.

As soon as you said `fieldset` I knew we'd hit this ancient Firefox bug from 2009!
https://bugzilla.mozilla.org/show_bug.cgi?id=504622

Your fix works great, but for Firefox only in v53 onwards.

There was a hacky workaround documented here:
https://thatemil.com/blog/2015/01/03/reset-your-fieldset/

```css
body:not(:-moz-handler-blocked) fieldset {
  display: table-cell;
}
```

But changing the `fieldset` display mode from _block_ might have accessibility implications.

### Firefox 52

![Firefox 52](https://user-images.githubusercontent.com/415517/55389160-9ba95a80-552c-11e9-8309-02152853de3c.png)

**Comment by kr8n3r** [comment]
@colinrotherham 😢 
yeah, haven't tried in in Firefox or widely in IE / Edge , only looked at older safari and chrome where min-width: fixes it.

I guess the fix depends on the support Frontend is looking to have (FF is currently 2 versions)

**Comment by 36degrees** [comment]
I believe this was addressed in #1269
